packages/strapi-generate-new/json/package.json.js

```diff
@@ -40,6 +40,7 @@ module.exports = scope => {
     'dependencies': {
       'lodash': '4.x.x',
       'strapi': getDependencyVersion(cliPkg, 'strapi'),
+      'strapi-upload-local': getDependencyVersion(cliPkg, 'strapi'),
```

Comment:
Should it be a dependence of `strapi-plugin-upload`?

Reply:
If you want to delete this provider, you can't if it's `strapi-plugin-upload` dependence.
Every `npm install` in `strapi-plugin-upload` will re-install this provider.